.github/workflows/create-backport-trackers.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           files: src/script/backport-create-issue src/script/requirements.backport-create-issue.txt
       - uses: actions/setup-python@v4
         with:
-          python-version: '>=3.6'
+          python-version: '>=3.6 <3.12'
```

Comment:
There's v4 out now?  Lemme guess -- more breaking changes?

Reply:
please disregard this